[release] v6.0.0-alpha.4 (#42005)
mui/material-ui: CHANGELOG.md

# [Versions](https://mui.com/versions/)

## v6.0.0-alpha.4

<!-- generated comparing v6.0.0-alpha.3..next -->

_Apr 23, 2024_

A big thanks to the 13 contributors who made this release possible. Here are some highlights ✨:

> - 🔥 Converted 3 more Material UI components to use Pigment CSS

Reviewer: ```suggestion
- 🔥 Converted 3 more Material UI components to use Pigment CSS.
```

> - ℹ️ Pigment CSS now lives in its own [repository](https://github.com/mui/pigment-css), all future work will be happening in the new location

Reviewer: ```suggestion
- ℹ️ Pigment CSS now lives in [its own repository](https://github.com/mui/pigment-css)! From now on, all future development will happen there.
```

Slight rephrasing and increasing the link target size

- [IconButton] Convert to support CSS extraction (#41850) @gijsbotje
- [Radio] Convert to support CSS extraction (#41840) @lhilgert9
- [Typography] Fix ownerState prop placement (#41903) @sai6855
- Generate typography tokens (#41703) @siriwatknp
- Move typography CSS variables to `font` (#42003) @siriwatknp
- Fix getOverlayAlpha type (#41995) @oliviertassinari
- Support CSS Extraction using codemod (#41935) @siriwatknp

### `@mui/icons-material@6.0.0-alpha.4`

- &#8203;<!-- 08 -->[icons] Update the icons package (#41937) @danilo-leal

### Docs

- [material-ui] Remove react-swipeable-views from demos as it's no longer maintained (#41912) @soler1212
- [material-ui] Add dark theme thumbnails for templates (#41947) @zanivan
- [material-ui] Remove links and interdependencies from free templates (#41941) @zanivan
- [material-ui] Add missing backticks to HTML tag in the installation page (#41972) @Miguelrom
- Fix 301 Toolpad links @oliviertassinari
- Fix 301 image redirections @oliviertassinari

### Core

- pnpm docs:zipRules && vale sync @oliviertassinari
- Remove @pigment-css/\* packages (#41965) @mnajdova
- [code-infra] Move the HighlightedCode component to @mui/docs (#41859) @Janpot

> - [code-infra] Move the HighlightedCode component to @mui/docs (#41859) @Janpot

Reviewer 2: @mnajdova not a biggie, but this line is there twice

Author: Thanks, I will update it, nice catch! :)

Reviewer 2: Don't want to inflate my contributions 😄
